client/src/components/LoginPage/InputField.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core";
+
+const styles = makeStyles({
+  textBox: {
+    margin: 10,
```

Comment:
Not required, but its generally better to use "rem" instead of pixels.  "rem" is alot more accessible as it will scale to the user's settings.  The default is 1rem = 16px, so for this it would be margin: "0.625rem"

Reply:
got it!